content/guide/_index.md

```diff
@@ -8,145 +8,207 @@ Welcome! This guide will help you get started with a Hugo Theme, including how t
+## Local development in a nutshell
 
- * `description` is for your default meta description, it will show on your homepage and other pages that have not set any meta description on this page front-matter.
+{{< youtube dyZcRRWiuuw >}}
 
- * `author` is also a meta tag. It belongs to the site author.
+**If the list above sounds like a foreign language to you, that’s okay! Let’s break it down:**
 
- * `mobile` is your default mobile number. If your theme has an option to make a call, then you can find it on default parameters.
+* **`Hugo` is a static site generator**. If you don’t have Hugo installed, you’ll need to [download](https://github.com/gohugoio/hugo/releases) and [install](https://gohugo.io/getting-started/installing/) it [make sure you have downloaded the [Hugo Extended](#install-hugo-extended-version) version for compiling scss files].
+* **A local server** is a temporary server environment on your computer-generated by your theme’s build tool. This avoids browser limitations of serving files directly from your computer’s filesystem (i.e. [CORS](https://developer.mozilla.org/en-US/docs/Glossary/CORS) limitations).
+* **Live reload** is a convenience function that many themes will include as part of their build tooling. It simply means that when you edit source files after they are recompiled, your browser will magically refresh and display the changes without a manual refresh.
 
- * `email` is your default email address. If your theme has an option to send an email, then you can find it on default parameters.
+## What are the source, compiled, and static files?
 
- * `address` is your default address. If your theme has an option to show your address, then you can find it on default parameters.
+These terms are can mean different things in different contexts, but for the purposes of a Hugo Theme:
 
- * `search` doesn't comes with every theme. But if your theme has a search bar, and if you don't want it, then you can disable it by making it `search = false`.
+* **Source files** are files that are meant to be processed by a theme’s build tools.
+* **Compiled files** are generated as a result of running a compiling process (also called a "build process") 
+on the source files.
+* **Static files** are ones that aren’t processed or generated.
 
- * `navigation_button` is for the main navigation button. Many themes have a button on navigation that target contact or other pages. If your theme has this button, then you can handle this button `label` and `link` from here, or if you don't want to show it, then you can disable it by `enable = false`.
+**Here are a few real-world examples of each type of file:**
+
+* **SCSS files always source files** because they must be compiled by your theme’s build tool to generate a CSS file that a browser can understand. SCSS files are located in `assets` folder.
```

Comment:
Codacy found an issue: [[maximum-line-length] Line must be at most 80 characters](https://app.codacy.com/gh/gethugothemes/gethugothemes-docs/pullRequest?prid=6304190)